### DISCUSSION TITLE
Is it possible to use this as the default editor for git?

### DISCUSSION THREAD

**Discussion by noah-nuebling** [coteditor/CotEditor#1158] [Q&A] [answered]
So that when you type 'git commit' this would come up and let you type the commit message.

**Comment by noah-nuebling** [comment, marked as answer]
It is possible! All you have to do is:

1. Install the CotEditor App from Mac App Store or somewhere else
2. Install the `cot` command line tool
    - `brew tap coteditor/coteditor`
    - `brew install cot`
3. Set it as you git default editor using the `--wait` argument
    - `git config --global core.editor 'cot --wait'`

That's it. Everything seems to work great!

**Reply by scottaw66** [reply]
Was cot removed from brew or renamed since this answer?

**Reply by noah-nuebling** [reply]
The cot command line tool is not in brew by default, it's in this repo: https://github.com/coteditor/homebrew-coteditor

You can add that repo to brew with the command `brew tap coteditor/coteditor`. Then you can install the command line tool with the command `brew install cot`.

But this doesn't install the CotEditor app itself. It only installs a command line tool which lets you interact with the CotEditor app from the command line.

To install the CotEditor app itself, you can download it from the App Store or somewhere else. It's also possible to install the app iself through Homebrew using the command `brew install --cask coteditor `

**Reply by scottaw66** [reply]
Ok, I deleted codeditor, re-installed using brew tap and then brew install cot worked fine.

Thank you.

**Reply by 1024jp** [reply]
Installing the `cot` command using homebrew is kind of deprecated and not recommended nowadays. Instead, use the `cot` command bundled in CotEditor.app itself. You can install it by following the instructions below:
http://coteditor.com/cot
